doc/conf.py

```diff
@@ -1,6 +1,10 @@
 import os
 import sys
 import datetime
+import locale
+
+# Otherwise VTK reader issues on some systems, causing sphinx to crash. See also #226.
+locale.setlocale(locale.LC_ALL, "en_US.UTF-8")
```

Comment:
Just so I understand this correctly: VTK IO takes into account locales, so reading a file from a different locale can break. Since our vtk files for the docs are guaranteed to be in the `en_US` locale, this needs to be done not to break doc building on any locale.

Reply:
Exactly, e.g. my own locale mixed up commas and dots, leading to the sphinx crash. Actually this is not really related to this PR (as it just broke doc building before the charts changes), but I forgot to open a separate issue for that. Should I do that instead and remove it from this PR?

Reply:
Nah, we (cough cough :laughing:) do that all the time with minor changes. Since this is confined to doc building and it just fixes an edge case bug, I think it's fine here.

Reply:
Good catch, because I completely missed this while debugging the sphinx build.